extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -216,8 +259,12 @@ export class SystemLayer {
     if (Services.viewerForDoc(this.win_.document.documentElement)
         .isEmbedded()) {
       this.systemLayerEl_.classList.add('i-amphtml-embedded');
+      this.getShadowRoot().setAttribute('info', '');
```

Comment:
Maybe we should add a `HAS_INFO_BUTTON_ATTRIBUTE` const to be consistent with the other attributes in this file